theano/tensor/opt.py

```diff
@@ -2747,6 +2797,7 @@ def local_subtensor_merge(node):
 @register_specialize
 @gof.local_optimizer([Subtensor])
 def local_subtensor_of_alloc(node):
+    #TODO Julian: Document this better!
```

Comment:
Do you want to address that during that PR?
Otherwise, I would remove the comment, as it may not make sense for someone else later.